src/websock/native/option.go

```diff
@@ -0,0 +1,21 @@
+package native
+
+import "time"
+
+// Option is used to configure the Handler
```

Comment:
How?

This is a particularly hard thing to describe in my experience, because the audience is both developers implementing options, and users wondering how the hell they find the actual options. I'm not sure if Godoc lists "constructor" functions for interfaces along with the interface, hopefully it does.

An attempt was made:

> Option apply some configuration change to a handler.
> Options are applied to the handler by passing them to NewHandler() .

Reply:
It's not the option's responsibility, but that's the key piece of information a user is going to need to see how they fit in. I'd favour documenting with the user as the audience here, since the whole reason for `Option` at all is to provide a better public API.